testing/trino-testing-containers/src/main/java/io/trino/testing/containers/BaseTestContainer.java

```diff
@@ -76,7 +83,8 @@ protected void setupContainer()
         for (int port : this.ports) {
             container.addExposedPort(port);
         }
-        filesToMount.forEach(this::copyFileToContainer);
+        resourcesToMount.forEach(this::copyFileToContainer);
```

Comment:
NIT: If we made that change so let's maybe rename copyFileToContainer to copyResourceToContainer